### PR TITLE
PR: Basic building test

### PR DESCRIPTION
**Pull request by rlaverde**
Fixes: #11

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+cookiecutter>=1.4.0
+
+# Testing
+pytest==3.0.5
+pytest-cookies==0.2.0
```
Comment (goanpeca) [inline requirements.txt:5]: Do you think we should have separate files for different requirements?

requirements_dev.txt
requirements_run.txt
??

Comment (rlaverde) [reply requirements.txt:5]: I don't think so, there are not a lot of requirements, I put this file just for documentation, and people using this template just will need to install `cookiecutter`

Comment (goanpeca) [reply requirements.txt:5]: Oops yes, I mean for the template (so the requirements inside the template)

Comment (rlaverde) [reply requirements.txt:5]: yes maybe a folder

```
requierements/
--base.py
--development.py
--testing.py
```

Comment (goanpeca) [reply requirements.txt:5]: .py ??? this is NOT DJANGO!! :wink:

Comment (rlaverde) [reply requirements.txt:5]: txt (and django doesn't use .py It was my error)

Comment (goanpeca) [reply requirements.txt:5]: 😆 messing with you :-p (actually I was thinking of setting.py!)

**Pull request merged**